rustls-mbedpki-provider/src/client_cert_verifier.rs

```diff
@@ -23,6 +25,8 @@ use crate::{
 pub struct MbedTlsClientCertVerifier {
     trusted_cas: mbedtls::alloc::List<mbedtls::x509::Certificate>,
     root_subjects: Vec<rustls::DistinguishedName>,
+    verify_callback: Option<Arc<dyn mbedtls::x509::VerifyCallback + 'static>>,
+    ignore_expired: bool,
```

Comment:
Can we add a type, something like `struct CertActiveCheck {pub ignore_expired: bool, pub ignore_not_active_yet: bool}`?

Reply:
make sense

Reply:
done!